jmeter/plans/find.rb

```diff
@@ -0,0 +1,137 @@
+require 'ruby-jmeter'
+
+BASEURL = ENV.fetch('JMETER_TARGET_BASEURL', 'http://localhost:3002')
+WAIT_FACTOR = ENV.fetch('JMETER_WAIT_FACTOR', 1).to_f
+THREAD_COUNT = ENV.fetch('JMETER_THREAD_COUNT', 200)
```

Comment:
Looking at the Find data on the day of the last spike, the concurrent user count estimate was ~150 at the peak hour. I've set a default count of 200 here, reflecting the estimate plus some additional margin.

This is far lower than the ~900 users we're testing in the Apply plan, but can be adjusted via the env var to test at that level as well if needed.

Reply:
Yea it seems very unlikely that apply will get 6 times the traffic Find got last year. Not that i think it hurts to be over prepared. Do you think it's worth ramping the load test up to say 300 if 200 causes no issues just to be safe?

Reply:
@mnacos just occurred to me that we'll need changes to how this is executed to support the new thread count env var. Is there anything else that needs to happen in code other than the Dockerfile update I just pushed?

Reply:
No, should be fine. You can pretty much define any env var you need via `cf set-env ...`.